dashboard/app/views/levels/editors/fields/_authored_hints.haml

```diff
@@ -67,6 +67,15 @@
             'markdown',
             {
               callback: function(editor, change) {
+                //Looking for `.markdown_preview div --> div --> [XML Blockly divs]` structure
+                //gets .markdown_preview div
+                let parent = space.find('.markdown_preview').get(0); 
+                //When XML is inserted, it creates a smaller div
+                let subParent = parent.querySelector("div");
+                //And then inside that div is where all of the Blockly XML divs are located that I want to delete
+                //If there aren't any divs: the querySelectors will gracefull fail without erroring because there's nothing to query or remove
+                subParent.querySelectorAll("div").forEach((e) => {e.remove()});
```

Comment:
Is it possible that this could remove `<div>` tags that are unrelated to the XML, and that we don't want to delete?

Reply:
Yes, this is also possible. I'm not sure how often `<div>` tags are included in authored hints but they are supported today:
![image](https://user-images.githubusercontent.com/43474485/236839841-cd8b0cdc-f09e-4c83-9784-1d89b2462f08.png)
This change would result in removing those elements from the markdown preview:
![image](https://user-images.githubusercontent.com/43474485/236839944-33be3ae3-b9a2-499c-bc57-2d2cf79a51b2.png)